lib/wallet/widgets/dialogs/your_ax_dialog.dart

```diff
@@ -4,199 +4,296 @@ import 'package:ax_dapp/wallet/widgets/ax_information_widgets.dart';
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 import 'package:url_launcher/url_launcher.dart';
+import 'package:flutter/foundation.dart';
```

Comment:
[nit:](https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak) Please sort directives alphabetically